Pocket Casts App Clip/NowPlayingView.swift

```diff
@@ -5,16 +5,36 @@ import StoreKit
 import PocketCastsUtils
 
 struct NowPlayingView: View {
+
+    enum ScreenState: Int {
+        case loading
+        case ready
+        case failed
+    }
+
     @State var presentAppStoreOverlay: Bool = false
+    @State var state: ScreenState = .loading
 
     var body: some View {
         VStack {
-            NowPlayingPlayerItemViewControllerRepresentable()
-                .onAppear {
-                    DispatchQueue.main.asyncAfter(deadline: .now() + 10) {
-                        presentAppStoreOverlay = true
+            Spacer()
+            HStack { Spacer() }
+            switch state {
+            case .loading:
+                ProgressView()
+                    .progressViewStyle(.circular)
+                    .tint(Color(uiColor: PlayerColorHelper.playerHighlightColor02()))
```

Comment:
As we discuss in our sync, the loader is too dark

Reply:
Done [here](https://github.com/Automattic/pocket-casts-ios/pull/2745/commits/76a520e5408a16b86e23eeafcaab74424c820881)